CHANGELOG.md

```diff
@@ -1,84 +1,173 @@
 
-## 1.11.0
-### Unreleased
+## 1.11.0-rc1
+### June 9, 2022
 
 CHANGES:
 
 * auth/aws: Add RoleSession to DisplayName when using assumeRole for authentication [[GH-14954](https://github.com/hashicorp/vault/pull/14954)]
 * auth: Remove support for legacy MFA
 (https://www.vaultproject.io/docs/v1.10.x/auth/mfa) [[GH-14869](https://github.com/hashicorp/vault/pull/14869)]
 * core: A request that fails path validation due to relative path check will now be responded to with a 400 rather than 500. [[GH-14328](https://github.com/hashicorp/vault/pull/14328)]
-* core: Bump Go version to 1.17.9. [[GH-go-ver-1110](https://github.com/hashicorp/vault/pull/go-ver-1110)]
+* core: Bump Go version to 1.17.11. [[GH-go-ver-1110](https://github.com/hashicorp/vault/pull/go-ver-1110)]
+* database/postgres: Change underlying driver library from [lib/pq](https://github.com/lib/pq) to [pgx](https://github.com/jackc/pgx) [[GH-15343](https://github.com/hashicorp/vault/pull/15343)]
```

Comment:
@kalafut do you think it's worth consolidating all these similar lines to reduce the total number of entries in the CHANGES section?

Reply:
Yeah, that makes sense to me. Maybe to something like:

```
* database & storage: Change underlying driver library.... This change affects Redshift & Postgres database secrets engines, and CockroachDB & Postgres storage engines.
```

Reply:
made the suggested changes